charts/timescaledb-single/templates/configmap-scripts.yaml

```
@@ -454,4 +454,5 @@ data:
         SELECT 'We are a primary: Successfully invoked checkpoints, now issuing a switchover.';
         \! curl -s http://localhost:8008/switchover -XPOST -d '{"leader": "$(hostname)"}'
     \endif
-...
+    -- If running with a linkerd-proxy, call /shutdown.
+    \! curl -X POST http://localhost:4191/shutdown || /bin/true
```

Should we add a timeout just in case it gets stuck on that? Otherwise there are chances that curl will wait indefinitely.
```suggestion
    \! curl -m 10 -X POST http://localhost:4191/shutdown || /bin/true
```

---

+1